kstreams-app/build.gradle

```diff
@@ -0,0 +1,80 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.15.1"
+        classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+    }
+}
+
+plugins {
+    id 'java'
+    id "com.google.cloud.tools.jib" version "1.5.0"
+}
+
+group = 'io.confluent'
+version = '0.0.8'
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+
+ext {
+  kafkaVersion = '2.6.0'
+  cpVersion = '6.0.0'
```

Comment:
can we leverage env_files/config.env now that this is in cp-demo?

Reply:
WDYT https://github.com/confluentinc/cp-demo/pull/258/commits/be73b0a20a04be3a8202c7e354afb80e5e691006 ?

Reply:
Slick!